### ISSUE TITLE
Can't install Requirements

### ISSUE THREAD

**Issue by brohan** [ahmedkhlief/APT-Hunter#7]
I installed python 3.9 on my Windows 10 Pro machine (only this fresh install of python, no other languages installed). When executing the command python -m pip install -r Requirements.txt I get the error listed below. I had run the command 2x, I assume this is why it says it references "cached"

Collecting netaddr
  Using cached netaddr-0.8.0-py2.py3-none-any.whl (1.9 MB)
Collecting evtx
  Using cached evtx-0.6.8.tar.gz (2.2 kB)
    ERROR: Command errored out with exit status 1:
     command: 'C:\Users\XXXXX\AppData\Local\Programs\Python\Python39\python.exe' -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\XXXXX\\AppData\\Local\\Temp\\pip-install-ehjf8erp\\evtx\\setup.py'"'"'; __file__='"'"'C:\\Users\\XXXXX\\AppData\\Local\\Temp\\pip-install-ehjf8erp\\evtx\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base 'C:\Users\XXXXX\AppData\Local\Temp\pip-pip-egg-info-yy_8dz2x'
         cwd: C:\Users\XXXXX\AppData\Local\Temp\pip-install-ehjf8erp\evtx\
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\XXXXX\AppData\Local\Temp\pip-install-ehjf8erp\evtx\setup.py", line 5, in <module>
        from setuptools_rust import RustExtension
    ModuleNotFoundError: No module named 'setuptools_rust'

Note that I am running the command in an elevated command prompt.

Thank you for any assistance.

**Comment by devgrips** [comment]
Hey, you just need to install setuptools-rust through pip.

**Comment by brohan** [comment]
Thank you I ran python pip install setuptools-rust, which appeared to run fine. In an elevated command prompt I re-ran the pip install requirements command and after a bit of time the message:

The system cannot execute the specified program

Any ideas what the hangup is? This is a domain joined Windows 10 20H2 computer, the command prompt elevation was granted via a local administrator account. I installed python directly from the python website, not through the Microsoft Store. I don't see Python listed in users\xxx\app data\ local\programs\ to edit any settings here.

Thank you again.
